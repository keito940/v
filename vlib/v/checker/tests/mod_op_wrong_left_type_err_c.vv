struct A{}
fn main() {
	a := A{}
	a % 1
}
