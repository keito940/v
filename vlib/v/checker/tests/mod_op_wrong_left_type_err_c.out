vlib/v/checker/tests/mod_op_wrong_left_type_err_c.v:4:2: error: mismatched types `A` and `int`
    2| fn main() {
    3|     a := A{}
    4|     a % 1
           ^
    5| }
